src/main/java/frc/robot/subsystems/BallPath/BallPathImpl.java

```diff
@@ -13,7 +18,32 @@
 // import frc.robot.subsystems.BallPath.Shooter.Shooter;
 // import frc.robot.subsystems.BallPath.Shooter.ShooterImpl;
 
-public class BallPathImpl {
-    
+public class BallPathImpl extends RepeatingPooledSubsystem implements BallPath {
+    public BallPathImpl() {
+        super(10, TimeUnit.MILLISECONDS);
+        // TODO Auto-generated constructor stub
+    }
+
+    // Declare interface with team
+        public void extendOuter(){};
+        public void retractOuter(){};
+        public void reverse(){};
+        // checks if ball is at bottom of elevator, checks sensors and primes
+        public boolean checkIfPrime(){
+            return true;
+        };
+        // can be used to stop a ball from going up the elevator in the event that we cannot shoot the ball
+        public void reverseElevator(){};
+        public void findAndCenterTarget(){};
+        public boolean readyToShoot(){
+            return true;
+        };
+        public void startShooter(){};
+        public void stopShooter(){};
+        public void defineResources(){};
+        public void task(){};
+        
+    @Override
+    public void lifecycleStatusChanged(LifecycleEvent previous, LifecycleEvent current) {};
```

Comment:
The semicolon after the curly brace is unnecessary on method definitions within a class, so they can be removed on all of these.